ext/opentelemetry-ext-wsgi/tests/test_wsgi_middleware.py

```diff
@@ -147,6 +147,20 @@ def test_wsgi_exc_info(self):
         response = app(self.environ, self.start_response)
         self.validate_response(response, error=ValueError)
 
+    def test_override_span_name(self):
+        """Test that span_names can be overwritten by our callback function."""
+        span_name = "Dymaxion"
+
+        def get_predefined_span_name(scope):
+            # pylint: disable=unused-argument
+            return span_name
```

Comment:
Nit: this could be replaced with a `Mock`:
```python
In [4]: from unittest.mock import Mock

In [5]: the_mock = Mock(**{"return_value": "Dymaxion"})

In [6]: the_mock({"a": "b"})
Out[6]: 'Dymaxion'
```